src/openms/include/OpenMS/SYSTEM/File.h

```diff
@@ -82,6 +82,18 @@ namespace OpenMS
     */
     static bool rename(const String& from, const String& to, bool overwrite_existing = true, bool verbose = true);
 
+    /**
+       @brief Copy directory recursively
+       
+       Copies a source directory to a new target directory (recursive).
+       
+       @param fromDir Source directory
+       @param toDir Target directory
+       @param overwrite_existing Delete existing target, before copying 
```

Comment:
it does not really delete the whole target, right? it just overwrites files which are also in the source.
Maybe that's worth adding to the docs.